test/tx-builder/dydx-governance.spec.ts

```diff
@@ -53,6 +58,27 @@ describeContract('DydxGovernance', init, (ctx: TestContext) => {
         );
         expect(emptyVoters.size).to.be.eq(0);
       });
+
+      it('getUserBalancesPerEpoch', async () => {
+        const userBalancesPerEpoch = await txBuilder.merkleDistributorService.getUserBalancesPerEpoch();
+        expect(Object.keys(userBalancesPerEpoch).length).to.be.eq(3);
+        expect(Object.keys(userBalancesPerEpoch['2']).length).to.be.eq(40867);
```

Comment:
I'm curious why indexing into the type `UserRewardsBalancesPerEpoch` with a key of type `string` is working, aren't the keys of type `number`? https://github.com/dydxfoundation/governance-contracts/blob/cd38076837f63c4909c0bf582d055b4941b5966c/src/tx-builder/types/GovernanceReturnTypes.ts#L128-L130

What happens if you write `        expect(Object.keys(userBalancesPerEpoch[2]).length).to.be.eq(40867);`

Reply:
yep - locally confirmed